components/application/Header.tsx

```diff
@@ -1,24 +1,51 @@
 import Component from "./Component";
-import Style from "./Header.module.scss";
-import {withTranslation, WithTranslation} from "next-i18next";
+import style from "./Header.module.scss";
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-class Header extends Component<HeaderProps, State> {
+const templateIds =  {
+  burgerButton: 'burgerButton'
+};
 
+export default class Header extends Component {
   public render() {
     return (
-      <div className={Style.Component}>
-        {this.props.t("Header_Title")}
-      </div>
+      <header className={`${style.header} d-md-flex mt-2`}>
+        <input type="checkbox" id={`${templateIds.burgerButton}`} className={`d-none`}></input>
+        <label className={`d-flex align-items-center justify-content-center my-3 ms-md-3`} htmlFor={templateIds.burgerButton}>
+          <h1 className="m-0 text-align-center">Website name</h1>
+          <div className={`${style.burgerMenu} d-md-none ms-3`}></div>
+        </label>
+        <nav className={`${style.navigation} d-md-block`}>
+          <ul className={`d-md-flex align-items-md-center ps-0 ps-md-2 m-auto`}>
+            {this.state.menuItems.map((menuItem, index) => 
+              <li className={`btn me-2 my-4 d-block`} key={index}>
+                <a className={`${style.menuLink} p-3`} href={menuItem.url}>{menuItem.titel}</a>
+              </li>
+            )}
+          </ul>
+        </nav>
+      </header>
     );
   }
+  state : {menuItems: IMenuItem[]} = {
```

Comment:
Normally you would create the state variable in the constructor of the component class.

Reply:
They have been moved.